cura/UI/PrintInformation.py

```diff
@@ -132,6 +134,14 @@ def setPreSliced(self, pre_sliced: bool) -> None:
             self._updateJobName()
             self.preSlicedChanged.emit()
 
+    @property
+    def slice_uuid(self) -> Optional[str]:
+        return self._slice_uuid
+
+    @slice_uuid.setter
+    def slice_uuid(self, value: Optional[str]) -> None:
+        self._slice_uuid = value
```

Comment:
Could consider making this a public field instead. Right now this is just overhead. That's what public fields are for, after all.

Reply:
Agreed I will change that in a minute